packages/video_player/video_player_android/android/src/test/java/io/flutter/plugins/videoplayer/VideoPlayerTest.java

```diff
@@ -278,4 +281,28 @@ public void onIsPlayingChangedSendsExpectedEvent() {
     assertEquals(event2.get("event"), "isPlayingStateUpdate");
     assertEquals(event2.get("isPlaying"), false);
   }
+
+  @Test
+  public void behindLiveWindowErrorResets() {
```

Comment:
Nit: can this be slightly more descriptive to describe what we expect? Something like:
```suggestion
  public void behindLiveWindowErrorResetsPlaybackPositionToDefault() {
```